<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="jenkins-agent-10-25-235-138" failures="0" tests="12" name="com.mypurecloud.sdk.v2.ApiClientRetryTest" time="73.023" errors="0" timestamp="2024-02-09T08:11:07 UTC" skipped="0">
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryTest" name="shouldRetryTestWith_502" time="1.015"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryTest" name="invokeTestWith_503" time="40.119"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryTest" name="shouldRetryTestWith_429" time="1.006"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryTest" name="shouldRetryTestWith_504_And_No_RetryAfter_Header" time="3.013"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryTest" name="invokeTestWith_429_And_No_MaxRetryTime" time="0.141"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryTest" name="invokeTestWith_429" time="6.522"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryTest" name="invokeTestWith_502" time="13.116"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryTest" name="invokeTestWith_504_No_MaxRetryTime" time="2.971"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryTest" name="invokeTestWith_504" time="2.101"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryTest" name="shouldRetryTestWith_200" time="0.006"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryTest" name="shouldRetryTestWith_503_And_RetryConfig" time="3.010"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryTest" name="shouldRetryTestWith_502_And_0_MaxRetryTime" time="0.003"/>
  <system-out/>
</testsuite> <!-- com.mypurecloud.sdk.v2.ApiClientRetryTest -->
